### **Q12.**

**(a)**

\begin{align*}
log(\frac{Pr(Y = orange | X = x)}{Pr(Y = apple | X = x)}) &= log(\frac{Pr(Y = orange | X = x)}{1 - Pr(Y = orange | X = x)}) \\ 

&= log(\frac{\frac{exp(\hat{\beta}_0 + \hat{\beta}_1 x)}{1 + exp(\hat{\beta}_0 + \hat{\beta}_1 x)}}{1 - \frac{exp(\hat{\beta}_0 + \hat{\beta}_1 x)}{1 + exp(\hat{\beta}_0 + \hat{\beta}_1 x)}}) \\

&= log(\frac{\frac{exp(\hat{\beta}_0 + \hat{\beta}_1 x)}{1 + exp(\hat{\beta}_0 + \hat{\beta}_1 x)}}{\frac{1}{1 + exp(\hat{\beta}_0 + \hat{\beta}_1 x)}})  \\ 

&= log(exp(\hat{\beta}_0 + \hat{\beta}_1 x)) \\

&= \hat{\beta}_0 + \hat{\beta}_1 x

\end{align*}

The log odds of orange versus apple in my model is $\hat{\beta}_0 + \hat{\beta}_1 x$

**(c)**
The given coefficients for our model:

\begin{align*}
\hat{\beta}_0 &= 2 \\ 

\hat{\beta}_1 &= -1
\end{align*}


By equating the log odds results from both models:

$$
\hat{\beta}_0 + \hat{\beta}_1 x = (\hat{\alpha}_{orange0} - \hat{\alpha}_{apple0}) + (\hat{\alpha}_{orange1} - \hat{\alpha}_{apple1}) x \\
$$

$$
\hat{\beta}_0 = (\hat{\alpha}_{orange0} - \hat{\alpha}_{apple0}) \\ 
$$

$$
\hat{\beta}_1 = (\hat{\alpha}_{orange1} - \hat{\alpha}_{apple1})
$$

By equating the log odds results from both models:

$$
\hat{\beta}_0 + \hat{\beta}_1 x = (\hat{\alpha}_{orange0} - \hat{\alpha}_{apple0}) + (\hat{\alpha}_{orange1} - \hat{\alpha}_{apple1}) x \\
$$

$$
\hat{\beta}_0 = (\hat{\alpha}_{orange0} - \hat{\alpha}_{apple0}) \\ 
$$

$$
\hat{\beta}_1 = (\hat{\alpha}_{orange1} - \hat{\alpha}_{apple1})
$$

From this we can see that infinitely many solutions exist, basically any two values with a difference equal to the estimated $\hat\beta$ coefficient would suffice.

$$
\begin{aligned}
\hat{\alpha}_{orange0} = 2 + c_0 \\
\hat{\alpha}_{apple0} = c_0 \\
\hat{\alpha}_{orange1} = -1 + c_1 \\
\hat{\alpha}_{apple1} = c_1 \\
\end{aligned}
$$

Where $c_0, c_1$ are constants. We can set these constants to 0 to eliminate redundancy and obtain the simplest case of:

$$
\begin{aligned}
\hat{\alpha}_{orange0} = 2  \\
\hat{\alpha}_{apple0} = 0 \\
\hat{\alpha}_{orange1} = -1 \\
\hat{\alpha}_{apple1} = 0 \\
\end{aligned}
$$

**(d)**
Note: There seems to be a mistake in the given coefficients of the question where it gives the coefficients for orange twice but not for apple so I'll assume the second set of coefficients is apple's.

$$
\begin{aligned}
\hat{\alpha}_{orange0} = 1.2 \\
\hat{\alpha}_{orange1} = -2 \\
\hat{\alpha}_{apple0} = 3 \\
\hat{\alpha}_{apple1} = 0.6 \\
\end{aligned}
$$

Using the same equations from above:

$$
\hat{\beta}_0 = (\hat{\alpha}_{orange0} - \hat{\alpha}_{apple0}) \\ 
$$

$$
\hat{\beta}_1 = (\hat{\alpha}_{orange1} - \hat{\alpha}_{apple1})
$$

In [3]:
beta0 = 1.2 - 3
beta1 = -2 - 0.6
beta0, beta1

(-1.8, -2.6)

The coefficient estimates for our model are

$$
\hat{\beta}_0 = -1.8 \\ 
$$

$$
\hat{\beta}_1 = -2.6 
$$

**(e)** I expect them to agree **every time** since the two models are equivalent and it's only a different coding of the same underlying logistic regression model.